### ISSUE TITLE
Khmer bold fonts

### ISSUE THREAD

**Issue by pnorman** [gravitystorm/openstreetmap-carto#362]
https://github.com/gravitystorm/openstreetmap-carto/pull/358#issuecomment-36010976 recommended the Khmer OS Kangrey for bold Khmer text, but I deferred because it wasn't packaged on Ubuntu, there are much bigger issues with Khmer text until we move to a version of Mapnik with Harfbuzz support (https://github.com/gravitystorm/openstreetmap-carto/pull/358#issuecomment-36101027) , and there is currently one Khmer label on the map, which doesn't use bold.

Once other font work is done, I want to revisit this, as I should have a better grasp on if other non-packaged fonts would be an improvement.
